spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializable.scala

```diff
@@ -232,41 +238,63 @@ trait ModuleSerializable extends Loadable with Savable{
       field.setAccessible(true)
       val fieldValue = field.get(module)
       DataConverter.setAttributeValue(context, attrBuilder, fieldValue, ptype)
-
       bigDLModelBuilder.putAttr(paramName, attrBuilder.build)
       })
   }
 
+  /*
+   * Re-create BigDL module by deserializing protobuf context.
+   * @param DeserializeContext: Deserialization context
+   * @param AbstractModule: The BigDL module to be re-created
+   * @return ModuleData: Tuple3 contains information of current module and modules adjacent to it
+   */
   protected def createBigDLModule[T: ClassTag](context: DeserializeContext,
                                                module : AbstractModule[Activity, Activity, T])
-                                              (implicit ev: TensorNumeric[T])
-  : ModuleData[T] = {
+                                              (implicit ev: TensorNumeric[T]): ModuleData[T] = {
     val model = context.bigdlModule
     val preModules = model.getPreModulesList.asScala
     val nextModules = model.getNextModulesList.asScala
     val bigDLModule = ModuleData(module, preModules, nextModules)
+
     if (model.getName != "") {
       module.setName(model.getName)
     }
+
     module.setNamePostfix(model.getNamePostfix)
+
     if (model.getTrain) {
       module.training()
     } else {
       module.evaluate()
     }
+
     module.inputShapeValue = ShapeConverter.shapeToBigDL(context, model, "input")
     module.outputShapeValue = ShapeConverter.shapeToBigDL(context, model, "output")
 
     // container does not need to be copied paramters again
     if (_copyWeightAndBias && context.bigdlModule.getSubModulesCount == 0) {
       copy2BigDL(context, bigDLModule)
     }
+
+    // Load MKL-DNN INT8 attributes (scales&mask of input&output) into
+    // BigDL Module from protobuf definition if the MKL-DNN INT8 flag is ON
+    if (model.getIsMklInt8Enabled) {
+      loadMklInt8Attr(context, module.asInstanceOf[MklInt8Convertible])
+    }
+
     bigDLModule
   }
 
-  protected def createSerializeBigDLModule[T: ClassTag](
-    modelBuilder : BigDLModule.Builder, context: SerializeContext[T])(implicit ev: TensorNumeric[T])
-  : SerializeResult = {
+
+  /*
+   * Create BigDL model's protobuf definition by serializing BigDL Module object
+   * @param BigDLModule.Builder: BigDL model builder of protobuf definition
+   * @param SerializeContext: Serialized context of BigDL module
+   * @return SerializeResult:
+   */
+  protected def createSerializeBigDLModule[T: ClassTag](modelBuilder : BigDLModule.Builder,
+                                      context: SerializeContext[T])
+                                    (implicit ev: TensorNumeric[T]): SerializeResult = {
```

Comment:
The syntax is a little strange.

Reply:
Thanks. I will modify these syntax issues.